index.bs

```diff
@@ -1388,6 +1389,12 @@ that are returned to the caller when a new credential is created, or a new asser
         {{CredentialsContainer/create()}}, this attribute's value will be an {{AuthenticatorAttestationResponse}}, otherwise,
         the {{PublicKeyCredential}} was created in response to {{CredentialsContainer/get()}}, and this attribute's value
         will be an {{AuthenticatorAssertionResponse}}.
+    
+    :   <dfn>authenticatorAttachment</dfn>
+    ::  This attribute reports the [=authenticator attachment modality=] in effect at the time this {{PublicKeyCredential}} was created.
```

Comment:
In the case of an authentication ceremony, I don't think we want to report the attachment modality when the credential was created.  We want the current attachment modality.

This mostly impacts caBLE authenticators.  where a credential created as platform can later be used as roaming.

In that case I don't think the client could know via CTAP2 how the credential was creates and would probably in an implementation report it as cross-platform in practice.    So I think this is more a problem with the wording of the descryption.

Reply:
Agreed. Current attachment modality is what is important.

Reply:
you're correct, resolving.

Reply:
hm, y'all are correct, I messed that wording up. dunno what I was thinking. @z11h will be proposing a fix.